hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -63,6 +64,9 @@ public void migrate(@NonNull final MigrationContext ctx) {
                 ctx.newStates().get(NODES_KEY);
         final var networkInfo = ctx.networkInfo();
         final var addressBook = networkInfo.addressBook();
+        final var bootstrapConfig = ctx.configuration().getConfigData(BootstrapConfig.class);
```

Comment:
We should validate this works fine with @tinker-michaelj 's upgrade test

Reply:
Merged with the latest develop branch. Ran DabEnabledUpgradeTest and passed.